great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -60,6 +60,15 @@ def __init__(
         self._ge_cloud_resource_type = ge_cloud_resource_type or singularize(
             ge_cloud_resource_name
         )
+
+        # TOTO: remove when account_id is deprecated
+        if ge_cloud_credentials.get("account_id"):
```

Comment:
Since we're not using the value here, could we use `if "account_id" in ge_cloud_credentials`? Very much a nit but a little bit more readable in my opinion

Reply:
Not in this case since that would just check for presence of key, but if key is there, we also want value to be not None.